dotnet-authserver/src/TeacherIdentity.AuthServer/Journeys/CoreSignInJourney.cs

```diff
@@ -29,16 +29,16 @@ public override bool CanAccessStep(string step)
     {
         return step switch
         {
-            Steps.Landing => !AuthenticationState.EmailAddressVerified,
-            SignInJourney.Steps.Email => !AuthenticationState.EmailAddressVerified,
-            SignInJourney.Steps.EmailConfirmation => AuthenticationState.EmailAddressSet && !AuthenticationState.EmailAddressVerified,
+            Steps.Landing => true,
```

Comment:
Slightly awkward that the back link on Email takes you to the landing page. Unfortunately at the moment we can't distinguish if we are just starting the journey or have returned from Check Answers (though we could without much trouble)